### ISSUE TITLE
about pretrained wavenet_vocoder?

### ISSUE THREAD

**Issue by cuicp** [NVIDIA/tacotron2#40]
I downloaded the latest code and trained 118 epochs,  and then downloaded the pre-trained wavnet vocoder(https://github.com/r9y9/wavenet_vocoder/) which trained with LJSpeech, but i could not synthesize audio similar to human voice.  Can you share which pre-trained wavnet vocoder  you used to synthesize the "demo.wav"? Thank you very much!

**Comment by yliess86** [comment]
Hi @cuicp, by any chance could I ask you what is your loss at epoch 118 ?

**Comment by rafaelvalle** [comment]
We used the WaveNet available at https://r9y9.github.io/wavenet_vocoder/
Note that you must adapt the mel output of Tacotron 2 to Ryuchi's mel representation.
Here are some basic guidelines:
```
# methods from from NVIDIAs repo
mel = torch.load(conditional_path)
mel = dynamic_range_decompression(mel)
mel = mel.cpu().numpy()
mel = mel.transpose()

# methods from from Ryuchi's repo
mel = audio._amp_to_db(mel) - hparams.ref_level_db
if not hparams.allow_clipping_in_normalization:
    assert mel.max() <= 0 and mel.min() - hparams.min_level_db >= 0
mel = audio._normalize(mel)
```

**Comment by deli666** [comment]
For generating audio sample with pre-trained wavenet_vocoder, do I need to apply the methods in both repos? or just apply in one of them?

**Comment by deli666** [comment]
Can you show more details about how to generate mel-spectrograms that can be synthesized by pre-trained wavenet_vocoder? I tried some methods but get only noises. The audio generated by Griffin-Lim sounds good.

**Comment by rafaelvalle** [comment]
@deli666 What exactly do you need?

**Comment by rafaelvalle** [comment]
@cuicp closing. please re-open if needed.
